gwswlib/hydx.py

```diff
@@ -8,6 +8,39 @@
 class Hydx:
     def __init__(self):
         self.connection_nodes = []
+        self.connections = []
+        self.structures = []
+
+    def import_csvline(self, line, csvfilename):
```

Comment:
Dit lijkt me op een verkeerde plek te zitten?
De keus wat je met een csv file moet doen hoef je maar één keer te maken: aan het begin v/h inlezen. Niet bij elke losse regel opnieuw.

Wat je zou kunnen doen is een `import_csvfile()` maken. Die kijkt wat het `hydxelement` moet zijn, importeert de regels en voegt ze toe aan connection_nodes/connections/structures/etc.